src/main/java/org/folio/service/userlookup/UserLookUp.java

```diff
@@ -0,0 +1,56 @@
+package org.folio.service.userlookup;
+
+import lombok.Builder;
+
+/**
+ * Retrieves user information from mod-users /users/{userId} endpoint.
+ */
+@Builder
+public class UserLookUp {
+  private String userName;
+  private String firstName;
+  private String middleName;
+  private String lastName;
+
+  /**
+   * Returns the username.
+   *
+   * @return UserName.
+   */
+  public String getUserName() {
```

Comment:
lombok can automatically generate getters.
any reason to explicitly defined them?

Reply:
fixed